### ISSUE TITLE
Variants with same chrom/pos/ref/alt but different transcripts?

### ISSUE THREAD

**Issue by dennishendriksen** [molgenis/molgenis-py-consensus#13]
How do the consensus scripts deal with variants with the same chrom-pos-ref-alt-gen that:
- lab 1 classifies with transcript X
- lab 2 classifies with transcript Y
?

**Comment by bartcharbon** [comment]
difference in transcript is ignored by design

right @marikaris ?

**Comment by marikaris** [comment]
> difference in transcript is ignored by design
> 
> right @marikaris ?

Correct. Transcript is not always delivered correctly and is usually not used by the labs when they classify a variant. Gene however is used in the classification.
